lino_extjs6/extjs6/linoweb.js

```diff
@@ -2108,6 +2107,7 @@ Lino.call_ajax_action = function(
     ,success: Lino.action_handler(panel, on_success, on_confirm)
     ,failure: Lino.ajax_error_handler(panel)
   });
+    if (panel.loadMask) panel.loadMask.hide();
```

Comment:
Yes , you are right. I fix it by hide the load message correctly in Lino.action_handler.